### ISSUE TITLE
[Docs] r-apt or r-rig or miniforge?

### ISSUE THREAD

**Issue by eitsupi** [rocker-org/devcontainer-features#74]
I think there needs to be documentation to show users the criteria for selection.

## memo

In the following case, that installing R and rmarkdown and jupyterlab on `ubuntu:jammy`, there is a twofold gap in build time on GitHub. (49.6s v.s. 97.7s)
https://github.com/rocker-org/devcontainer-features/actions/runs/3258663254/jobs/5350921787

```json
"features": {
    "ghcr.io/rocker-org/devcontainer-features/r-apt:0": {
        "installRMarkdown": true,
        "installJupyterlab": true
    }
}
```

**Comment by eitsupi** [comment]
> I installed r-cran-dplyr on Debian 11 bullseye today and ran into a bug in dplyr 1.0.4 and ended up having to source install dplyr from CRAN......

It seems worth trying to install from testing.

**Comment by eitsupi** [comment]
Closed by rocker-org/website#84
